### ISSUE TITLE
Current default bosh azure cpi + stemcell violates the release notes requirements and does not deploy

### ISSUE THREAD

**Issue by haydonryan** [enaml-ops/omg-cli#8]
```
./omg-linux azure \
  --azure-vnet boshvnet-crp \
  --azure-subnet Bosh \
  --azure-subscription-id <Redacted> \
  --azure-tenant-id <Redacted>  \
  --azure-client-id <Redacted> \
  --azure-client-secret amazon4ever \
  --azure-resource-group hrcentral \
  --azure-storage-account <Redacted> \
  --azure-security-group nsg-bosh \
  --azure-ssh-pub-key-path bosh-privatekey.pub \
  --azure-ssh-user vcap \
  --azure-environment AzureCloud \
  --azure-private-key-path bosh-privatekey \
  --cpi-name cpi \
  --bosh-private-ip 10.0.0.5 \
  --print-manifest

```

then remove the public IP address attempts to deploy cpi v11 and azure stemcell 3169 which is < 3181 as per below:

Fixes: - Use LUN and host device id as the disk identifier - Compatible Stemcell Versions: v3181 or later - Stemcell v3232.5 or later is recommended with CPI v11. You may hit the issue #135 if you still use an older stemcell than v3232.5 - Use a data disk as the persistent ephemeral disk - The VM size should not be Standard_A0 if you want to use persistent data disks - The CPU cores of VM should be more than 1 if you plan to resize the persistent data disks in future - The premium storage is recommended for better performance - You can see more information about the ‘Max data disks’ and ‘Max disk IOPS & bandwidth’ of Azure Linux VMs here - Fix an issue in calculating the sleep interval when copying blobs
[http://bosh.io/releases/github.com/cloudfoundry-incubator/bosh-azure-cpi-release?version=11](http://bosh.io/releases/github.com/cloudfoundry-incubator/bosh-azure-cpi-release?version=11)

**Comment by calebwashburn** [comment]
Please re-validate on latest release v0.0.10

**Comment by haydonryan** [comment]
Still an issue. CPI V11+  requires 3181 or later.  default in omg 0.0.10 is 3169.

**Comment by calebwashburn** [comment]
Updated defaults to use latest stemcell version for azure.  https://github.com/enaml-ops/omg-cli/releases/tag/untagged-542d80a2f8437855ebc0.  Will be included in next release v0.0.11
